### PR TITLE
Support `freezed` 1.0

### PR DESCRIPTION
**Pull request by AlexV525**

### CODE REVIEW

**Comment by AlexV525** [comment]
I'm not sure whether `freezed` files need to be updated too. If not, I can make a rebase.

File: packages/stripe_platform_interface/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: stripe_platform_interface
 description: Platform interface for stripe sdk
-version: 2.0.2
+version: 2.0.3
```
Comment (remonh87) [inline packages/stripe_platform_interface/pubspec.yaml:3]: Can you revert this? Version bump is not needed we use Melos to bump and publish the versions

**Review by remonh87 — CHANGES_REQUESTED** [review]
Thanks for creating this PR! IF you can revert the version bump in the pub spec yaml. We are good to go

**Review by remonh87 — APPROVED** [review]

**Comment by AlexV525** [comment]
Publish?

**Comment by AlexV525** [comment]
@remonh87 Please publish this when you have time for it, we've been blocked from it for nearly a month.

**Comment by remonh87** [comment]
@AlexV525  we want to incorporate two other features in this release. There is no need to be blocked you can add an override in your app. I did this for my project as well. We will publish this fix before Christmas for sure (whether or not the 2 other features are implemented).

`dependency_overrides:   freezed_annotation: ^1.0.0`

**Comment by AlexV525** [comment]
We can make an override for sure, it's all about consistency.

**Pull request merged**